components/navbar/NotificationBoxButton.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <a class="navbar-item" @click.stop="toggleNotificationModal">
+    <span v-if="props.showLabel">{{ $t('notification.notifications') }}</span>
+    <NeoIcon ref="root" icon="bell" class="icon" />
+  </a>
+</template>
+
+<script setup lang="ts">
+import { NeoIcon } from '@kodadot1/brick'
+import { NotificationBoxModalConfig } from '@/components/common/NotificationBox/useNotificationBox'
+import { BModalComponent, BModalConfig } from 'buefy/types/components'
```

Comment:
Have you tried to use [oruga modal](https://oruga.io/components/modal.html)?

Reply:
or use a drawer ([Sidebar](https://oruga.io/components/Sidebar.html)) component instead

Reply:
https://github.com/kodadot/nft-gallery/pull/5369#issuecomment-1493794515